classes/quiz_settings.php

```diff
@@ -380,18 +366,23 @@ private function process_bool_settings() {
     /**
      * Turn hashed quit password and quit link into PList strings and add to config PList.
      */
-    private function process_quit_settings() {
+    private function process_quit_password_settings() {
         $settings = $this->to_record();
         if (!empty($settings->quitpassword) && is_string($settings->quitpassword)) {
             // Hash quit password.
             $hashedpassword = hash('SHA256', $settings->quitpassword);
             $this->plist->add_element_to_root('hashedQuitPassword', new CFString($hashedpassword));
         }
+    }
 
+    private function process_quit_url_from_form() {
```

Comment:
How about to rename this method to get rid of the form as form doesn't make sense in this context. Something like process_quit_url_manual or process_quit_url_from_settings

Reply:
This has been renamed to `process_quit_url_from_settings`